packages/contracts/src/deploy.ts

```diff
@@ -1,132 +0,0 @@
-import { ethers } from 'ethers'
```

Comment:
`unirep-social` and the backend rely on this script. Can we do this refactor in a way that isn't breaking? Or at least leave this file in place until pr's are through in the other repos?

Reply:
Ideally I think backend should not reply on the script (because the verifiers are related to circuits)
It is also a part of making packages independent of circuits
so I think maybe we can upload circuits/verifiers to a server then the unirep-social can use the desired circuits/verifiers
unirep-social should deploy the verifiers first then deploy `Unirep.sol` in `@unirep/contracts`

but yeah we can still keep this `deploy.ts` until unirep-social is completed.